packages/sdks/web-js-sdk/src/enhancers/withPersistTokens/helpers.ts

```diff
@@ -67,7 +73,13 @@ export const persistTokens = (
   // persist session token
   if (sessionJwt) {
     sessionTokenViaCookie
-      ? setJwtTokenCookie(SESSION_TOKEN_KEY, sessionJwt, cookieParams)
+      ? setJwtTokenCookie(SESSION_TOKEN_KEY, sessionJwt, {
+          ...cookieParams,
+          cookieSameSite:
+            sessionTokenViaCookie === true
```

Comment:
if someone will pass invalid value (e.g. `same-site` instead `SameSite` or just `{}`, they can do it even though TS allow this) - the fallback here would be browser's - which might be lax

there could be a theoretical BC issue, if someone passed before `sessionTokenViaCookie={'true'}` or  `sessionTokenViaCookie={{}}`

I would do something more stable such as `sessionTokenViaCookie?.SameSite || 'Strict'`